lnrpc/README.md

```diff
@@ -146,6 +146,16 @@ $ go get -u github.com/lightningnetwork/lnd/lnrpc
 
 ## Generate protobuf definitions
 
+### Linux
+
+For linux there is an easy install script that is also used for the Travis CI
```

Comment:
Note that the script only works for Debian/Ubuntu and does not check if there's a preinstalled toolset. For those users (and in general) a hermetic, reproducible build system like Bazel may be a better choice.

Reply:
Yes, that's true. But quite a few users have struggled with this doing it manually. So having a script already helps a lot in my opinion. I added a description what tools need to be installed to make it more clear that this is not an all-in-one solution.

Reply:
Sorry, didn't articulate really what I meant. My main point is that as this script applies to Debian/Ubuntu only, I'd specifically say that instead of referring to all "linux" users.

Reply:
Which part is Ubuntu specific? The `apt-get` stuff is only in `.travis.yml`. The bash script should work on most systems.

Reply:
Yeah, sorry. Somehow I remembered that part to be in the `installl_travis_proto.sh` but just checked and it's indeed in `travis.yml`.